build.sbt

```diff
@@ -83,15 +83,6 @@ lazy val docs = project
     publishRsyncHost := "akkarepo@gustav.akka.io",
     apidocRootPackage := "akka")
 
-Global / onLoad := (Global / onLoad).value.andThen { s =>
-  val v = version.value
-//  TODO commented because it does not compile
-//  if (dynverGitDescribeOutput.value.hasNoTags)
-//    throw new MessageOnlyException(
-//      s"Failed to derive version from git tags. Maybe run `git fetch --unshallow`? Derived version: $v")
-  s
-}
-
```

Comment:
is it not necessary to proper working of sbt-dynver? Is it working correctly now?

Reply:
It is not. As described during the call, sbt-dynver handles the case with missing tags.